src/processor.ts

```diff
@@ -74,6 +74,7 @@ export interface Processor {
   setup?: (p: ReturnType<typeof createSquidProcessor>, chain: Chain) => void
   process: (ctx: Context) => Promise<void>
 }
+export const createProcessor = (p: Processor) => p
```

Comment:
same than previous `gql` helper, why those identity functions?

Reply:
I often find myself writing processors and having to remind myself what the schema is. if I just call this function it helps me to know. Otherwise you could do `const armProcessor: Processor = { /* yada yada */ }`

Why this over that? Good question I guess.